panoramapublic/src/org/labkey/panoramapublic/query/ExperimentAnnotationsTableInfo.java

```diff
@@ -225,7 +225,7 @@ public void renderGridCellContents(RenderContext ctx, Writer out) throws IOExcep
         runCountSQL.append(" WHERE r.ExperimentId = ");
         runCountSQL.append(ExprColumn.STR_TABLE_ALIAS);
         runCountSQL.append(".ExperimentId)");
-        ExprColumn runCountColumn = new ExprColumn(this, "Runs", runCountSQL, JdbcType.INTEGER);
+        ExprColumn runCountColumn = new ExprColumn(this, "Skyline Docs", runCountSQL, JdbcType.INTEGER);
```

Comment:
We generally try to avoid column names with spaces. For maximum compatibility, I recommend keeping the column name as-is but setting the label/caption to Skyline Docs.